### PR TITLE
Add CfgMgmt CFP to website

### PR DESCRIPTION
**Pull request by GregSutcliffe**
1 week left on the CFP, figured it should get a push :)

### CODE REVIEW

File: _includes/alerts.html [hunk 1]
```diff
@@ -1,5 +1,8 @@
-<!-- <div class='tease'>      -->
-<!--   <p>                    -->
-<!--     something important! -->
-<!--   </p>                   -->
-<!-- </div>                   -->
+<div class='tease'>
+  <p>
+  <a>Conference season is here! Join us at FOSDEM &amp; Config Management Camp, 2nd - 5th February 2019</a>
```
Comment (tbrisker) [inline _includes/alerts.html]: Shouldn't this be 6th?

Comment (GregSutcliffe) [reply _includes/alerts.html]: I guess that's the hackday rather than part of the conference proper, but I can amend if you think it's right.

Comment (GregSutcliffe) [reply _includes/alerts.html]: @tbrisker updated

**Review by tbrisker — APPROVED** [review]
Thanks!

**Pull request merged**